assets/src/edit-story/app/media/utils/useProcessVideo.js

```diff
@@ -108,6 +139,17 @@ function useProcessVideo({
         }
       };
 
+      const onUploadProgress = ({ resource }) => {
+        const oldResourceWithId = { ...resource, id: oldResource.id };
+        updateElementDimensions({
+          id: oldResource.id,
+          resource: oldResourceWithId,
+        });
+        updateExistingElements({
+          oldResource: oldResourceWithId,
+        });
```

Comment:
Updating the resource like this, means src is set empty, making it not playable, which is what we want.